docs/install/docker.md

```diff
@@ -27,6 +27,9 @@ cp docker-compose.override.yml-example docker-compose.override.yml
 vim docker-compose.override.yml
 ```
 
+Next you may customize your [environment configuration](/docs/customization/configuration.md#environment-configuration) if you choose, but it should work
```

Comment:
Hmmm. After taking a look at this whole document, it could use a bit of cleanup. I think the initial section `First Time Installation` should omit all information about customizing the environment and docker-compose file. I think simply mentioning that the Docker compose file will work with zero configuration is adequate, as that was our goal. It'd be friendlier if it explicitly had the `git clone` commands for whether one wants to develop Open ONI (`dev` branch) or build a site with it (`master`/`main` branch). And then close that first section by providing a more convenient URL directly to `http://localhost`. This should be the easiest, simplest part to just get Docker running ONI and returning a web page.

I think all discussion of the Docker Composer override and `.env` file should be moved to the `Overriding URL and Ports` section, and then I'd rename it a little more broadly to `Customizing the Docker Environment` or similar. Mention again that no customization is needed, but maybe suggest the most common changes like the URL and ports. The overrides file isn't necessary, but will likely be helpful for people developing on ONI itself. Probably less so for those building sites on top of it.

@jscancella @jechols, I hope this addresses the conversation elsewhere about the overrides file and makes sense in terms of re-organization. Let me know what you think.

Reply:
@techgique please review the updated documentation as I believe it addresses all your helpful feedback.